src/OHIFStandaloneViewer.js

```diff
@@ -79,24 +87,84 @@ class OHIFStandaloneViewer extends Component {
       );
     }
 
+    /**
+     * Note: this approach for routing is caused by the conflict between
+     * react-transition-group and react-router's <Switch> component.
```

Comment:
Hmm... What exactly was the issue?

“Routing transition effect for React — with react-router v4 and react-transition-group” by Hyeokwoo Alex Kwon https://link.medium.com/2IU3mc8jRX

I honestly don't know enough to weigh in on the removal of `<switch>`, but it does look like we can keep it.

Reply:
Yeah I did it with Switch like that initially and everything appeared to work, but the window resize handlers would start being called in infinite loops with short delays for no clear reason. 

The Cornerstone element started off super small and would grow and grow forever. I tried to debug it for a while and eventually gave up on that approach. This is what react-transition-group recommends in their docs (https://reactcommunity.org/react-transition-group/with-react-router) so that's good enough for me.